charts/enterprise-metrics/templates/graphite-write-proxy-dep.yaml

```diff
@@ -0,0 +1,130 @@
+{{- if .Values.graphite.enabled -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "enterprise-metrics.fullname" . }}-graphite-write-proxy
+  labels:
+    app: {{ template "enterprise-metrics.name" . }}-graphite-write-proxy
+    chart: {{ template "enterprise-metrics.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+  annotations:
+    {{- toYaml .Values.graphite_write_proxy.annotations | nindent 4 }}
+spec:
+  replicas: {{ .Values.graphite_write_proxy.replicas }}
+  selector:
+    matchLabels:
+      app: {{ template "enterprise-metrics.name" . }}-graphite-write-proxy
+      release: {{ .Release.Name }}
+  strategy:
+    {{- toYaml .Values.graphite_write_proxy.strategy | nindent 4 }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "enterprise-metrics.name" . }}-graphite-write-proxy
+        # The name label is important for cortex-mixin compatibility which expects certain names for services.
+        name: graphite-querier
+        target: querier
```

Comment:
i think those values are not correct

Reply:
In fact they shouldn't be there at all, good catch. They are for the cortex mixin which is not supporting graphite at the moment so I'm removing them from both the querier and the write proxy. https://github.com/grafana/helm-charts/pull/872/commits/a0a682ed399a306223b01dcc00f8d5785a719046